packages/versioning/src/lib.rs

```diff
@@ -1,115 +1,88 @@
-use serde::{Deserialize, Serialize};
-
 use sdk::{
-    cosmwasm_std::{to_binary, Binary, StdError, StdResult, Storage},
+    cosmwasm_std::{StdError, StdResult, Storage},
     cw_storage_plus::Item,
-    schemars::{self, JsonSchema},
 };
 
 pub type Version = u16;
 
-pub const VERSION_ITEM: Item<'static, Version> = Item::new("contract_version");
+pub type SemVer = (Version, Version, Version);
+
+#[macro_export]
+macro_rules! package_version {
```

Comment:
why do we define it as a macro instead of as a regular function?

Reply:
wny not just passing core::env!("CARGO_PKG_VERSION")?

Reply:
My point was to not have any constants and stuff lying around and at the same time making it more ergonomic to use.